CMakeLists.txt

```diff
@@ -54,27 +50,30 @@ add_library(witmotion_ros
 
 message(WARNING "tf2_geometry_msgs include dir: ${tf2_geometry_msgs_INCLUDE_DIRS} ")
 
-#find_package(Qt5 REQUIRED COMPONENTS Core SerialPort)
-set(INCLUDE_DIRS ${ament_cmake_INCLUDE_DIRS} ${tf2_INCLUDE_DIRS}
```

Comment:
change CMAKE style dependency management to more idiomatic `ament_target_dependencies`